Src/Support/IntegrationTests/IntegrationTests.Utils/Google.Apis.IAMCredentials.v1.cs

```diff
@@ -0,0 +1,739 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Generated code. DO NOT EDIT!
+
+namespace Google.Apis.IAMCredentials.v1
+{
+    /// <summary>The IAMCredentials Service.</summary>
+    public class IAMCredentialsService : Google.Apis.Services.BaseClientService
+    {
+        /// <summary>The API version.</summary>
+        public const string Version = "v1";
+
+        /// <summary>The discovery version used to generate this service.</summary>
+        public static Google.Apis.Discovery.DiscoveryVersion DiscoveryVersionUsed = Google.Apis.Discovery.DiscoveryVersion.Version_1_0;
+
+        /// <summary>Constructs a new service.</summary>
+        public IAMCredentialsService() : this(new Google.Apis.Services.BaseClientService.Initializer())
+        {
+        }
+
+        /// <summary>Constructs a new service.</summary>
+        /// <param name="initializer">The service initializer.</param>
+        public IAMCredentialsService(Google.Apis.Services.BaseClientService.Initializer initializer) : base(initializer)
+        {
+            Projects = new ProjectsResource(this);
+        }
+
+        /// <summary>Gets the service supported features.</summary>
+        public override System.Collections.Generic.IList<string> Features => new string[0];
+
+        /// <summary>Gets the service name.</summary>
+        public override string Name => "iamcredentials";
+
+        /// <summary>Gets the service base URI.</summary>
+        public override string BaseUri =>
```

Comment:
Once we've updated to the next generator version, we'll no longer generate the conditional part, FWIW. Might be worth adapting to that now, given that we're not targeting those platforms for tests anyway.

Reply:
(Or maybe just leave a note somewhere to remind ourselves to update it later.)

Reply:
I've filed #2265